cves/CVE-2013-6658.yml

```
@@ -121,10 +121,10 @@ interesting_commits:
     emerging themes?
 
     If there are no interesting commits, demonstrate that you completed this section by explaining what happened between the VCCs and the fix.
-  answer:
+  answer: one interesting commit was found.
   commits:
-  - commit: 
-    note: 
+  - commit: 5a430e63a4c764997a1e6612ed1f723acce94461 
+    note: in this commit, the author notes in the commit message that this area where the destroy() method is used, is a very risky area in terms of memory allocation. This indicates that the vulnerability could have been avoided as it has been caught between VCC and fix.
```

```suggestion
    note: In this commit, the author notes in the commit message that this area where the destroy() method is used, is a very risky area in terms of memory allocation. This indicates that the vulnerability could have been avoided as it has been caught between VCC and fix.
```

Make sure to include some of this information in the answer tag above.

---

Thanks good catch